tensorflow/compiler/tests/binary_ops_test.py

```diff
@@ -20,6 +20,7 @@
 
 import numpy as np
 
+import tensorflow as tf
```

Comment:
Instead of this, this test should reach into where sparse matmul is originally defined and use it. Here is an example:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/sparse_matmul_op_test.py#L54